### ISSUE TITLE
Have a python3 windows test run on Appveyor

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8678]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8678|
|Type|enhancement|
|Created|2016-07-25 16:29:49Z|

it can fail... but hope it can help with future port work 

it can be python 3.5 only as a start

<details><summary>Searchable metadata</summary>

```
trac-id__8678 8678
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469464189799615 1469464189799615
changetime__1471040267761561 1471040267761561
version__None None
owner__None None

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

In changeset ef6964a725673632dcb3ccd505c9d8b344607d3d
```
#!CommitTicketReference repository="" revision="ef6964a725673632dcb3ccd505c9d8b344607d3d"
Add topfile for #8678
```

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set status to `closed`|
|-|-|

Fixed in [94f96916](https://github.com/twisted/twisted/commit/94f969166dbdb7a74c6888e452b5ab2c739a9376)
